examples/math/trigonometryExample/src/ofApp.cpp

```diff
@@ -35,27 +35,26 @@ void ofApp::setup(){
 
 //--------------------------------------------------------------
 void ofApp::update(){
-	cout << rotationSpeed << endl;
     
     if (!ofGetMousePressed()) {//press the mouse to stop the angle from incrementing.
 		angle+=rotationSpeed;//at each update the angle get's incremented 
 	}
-	if (angle>=TWO_PI) { //if the angle is more than or equal to two PI (a full rotation measured in Radians) then make it zero.
-		angle=0;
-	}
-
+	 //if the angle is more than or equal to two PI (a full rotation measured in Radians) then make it zero.
+	angle = ofWrap(angle, 0, TWO_PI);
```

Comment:
This still isn't fixed.  I'll fix it in a PR.

Reply:
this is fine. What was there before was doing the same but only for when the angle was more than a full rotation but it was not handling the case when it was less than zero, which it is now.

Reply:
OK.  Reverting in my PR.